Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Views/Selectors.cs

```diff
@@ -88,6 +94,20 @@ public static int GetSelectedBindingIndexAfterCompositeBindings(InputActionsEdit
             return state.selectedBindingIndex + toSkip;
         }
 
+        public static int GetBindingIndexBeforeAction(SerializedProperty arrayProperty, int indexToInsert, SerializedProperty bindingArrayToInsertTo)
+        {
+            var offset = 1; //previous action offset
```

Comment:
Should there be at least a Debug.Assert here on expected range of indexToInsert? Currently its used whatever it is, e.g. -21

Reply:
Might help catch errors later...

Reply:
fixed in